src/javascript/note.js

```diff
@@ -73,12 +72,14 @@ export default class Note {
     this.judge();
```

Comment:
Wait... what? Is this `this.judge()` recursively calling itself?

Reply:
??!!!?!

Reply:
You scared me for a sec... That code was collapsed :D